tools/fastg2protlib/fastg2protlib-peptides.xml

```diff
@@ -25,12 +23,11 @@
         </param>
         <param name="min_protein_length" type="integer" value="55" label="Minimum Protein Length in Amino Acids" />
         <param name="min_peptide_length" type="integer" value="8" label="Minimum Peptide Length in Amino Acids" />
-        <param name="database_name" type="text" value="results.db" label="Database Name" />
         <param name="show_plots" type="boolean" checked="true" label="Create Diagnostic Plots" />
     </inputs>
     <outputs>
         <data name="peptide_fasta" format="txt" from_work_dir="peptide.fasta" label="Peptides from FASTG" />
```

Comment:
Can you prefix also the other output labels with `${on_string}`

Reply:
Done